apps/hrea-graphql-explorer/package.json

```diff
@@ -24,7 +24,7 @@
     "rimraf": "^3.0.2"
   },
   "scripts": {
-    "start": "REACT_APP_HC_APP_ID='hrea_suite' REACT_APP_HC_CONN_URL='ws://localhost:4000' BROWSER=none react-scripts start",
+    "start": "REACT_APP_HC_APP_ID='hrea_suite' REACT_APP_HC_CONN_URL='ws://localhost:4000' REACT_APP_HC_ADMIN_CONN_URL='ws://localhost:55355' BROWSER=none react-scripts start",
```

Comment:
I can't see in `run-dev-conductor.sh` that I am going to be able to make the admin websocket the same every time. It doesn't seem to be an option. It randomly picks one.

Reply:
@pospi I did get the GraphQL explorer working again, but you have edit and set this manually for now, based on whatever `npm run dht:conductor` spits out